ruby_on_rails/rails_sprinkles/css_bundling.md

```diff
@@ -56,7 +56,7 @@ That's all there is to it.
 
 When you use cssbundling it does a couple of things for you. Firstly, no matter which option you choose, it installs the tools and files necessary to get it running and ensures the required import statements are added to your application. It then gives you instructions on the build script you need to use.
 
-It's interesting to see how simple this gem is. If you see the [lib/install](https://github.com/rails/cssbundling-rails/tree/main/lib/install) directory for the gem you'll notice how few files there are. The `install.rb` file handles making sure all the necessary parts are in place to handle whichever install option you use, and then each tool has its own directory with the files required for that tool to work. Have a look at the [Bootstrap](https://github.com/rails/cssbundling-rails/blob/main/lib/install/bootstrap/install.rb) install file. It's only 27 lines long.
+It's interesting to see how simple this gem is. If you see the [lib/install](https://github.com/rails/cssbundling-rails/tree/main/lib/install) directory for the gem you'll notice how few files there are. The `install.rb` file handles making sure all the necessary parts are in place to handle whichever install option you use, and then each tool has its own directory with the files required for that tool to work. Have a look at the [Bootstrap](https://github.com/rails/cssbundling-rails/blob/main/lib/install/bootstrap/install.rb) install file. It's only 32 lines long.
```

Comment:
I think it's probably not a great idea for the lesson to refer to the specific line length of the file. This will always be subject to change, and any time it does change, what we say in the lesson will be inaccurate.

Maybe this can just be something like:
```suggestion
It's interesting to see how simple this gem is. If you see the [lib/install](https://github.com/rails/cssbundling-rails/tree/main/lib/install) directory for the gem you'll notice how few files there are. The `install.rb` file handles making sure all the necessary parts are in place to handle whichever install option you use, and then each tool has its own directory with the files required for that tool to work. Have a look at the [Bootstrap](https://github.com/rails/cssbundling-rails/blob/main/lib/install/bootstrap/install.rb) install file. Notice how short it is.
```

Reply:
Yes, i totally agree with your change

Reply:
Awesome. Feel free to make a new commit to this PR that adds that change in. Or you could click the "Commit suggestion" button that appears under my suggested change. Either way is fine with me.

Reply:
Hi mates, sorry for the delay, i've just make a new commit with the apropriate changes.